### DISCUSSION TITLE
Releases build branch

### DISCUSSION THREAD

**Discussion by cjundt** [ClosedXML/ClosedXML#1955] [Q&A] [answered]
Hi,

On which branch (maser, develop or release-candidate) do you build releases ?
I must build my own pre-release version but I still want to be able to catch up next release when it available.

Yours,
Christian

**Comment by jahav** [comment, marked as answer]
Use develop, that is the most up to date with commits for a release. The release build is taken from a build made by appveyor from the release tag.

I will update release-candidate and master before release (forgot last time). Other branches are there if there was divergent development for release and *develop* had ongoing development that shouldn't be a part of the release.

**Reply by cjundt** [reply]
Thanks
